sci-libs/vtk/files/vtk-9.1.0-avoid-naming-collision-with-netcdf-4.9.0.patch

```diff
@@ -0,0 +1,24 @@
+https://bugs.gentoo.org/851594
+
+From 6bb7a11d69c87d4e9958b641c5d32e27bc933389 Mon Sep 17 00:00:00 2001
+From: Bernd Waibel <waebbl-gentoo@posteo.net>
+Date: Sat, 18 Jun 2022 12:23:02 +0200
+Subject: [PATCH] avoid naming collision with netcdf-4.9.0
+
+The identifier has been #defined with netcdf-4.9.0 already. To avoid
+conflicts remove the declaration.
+
+Signed-off-by: Bernd Waibel <waebbl-gentoo@posteo.net>
+--- a/ThirdParty/exodusII/vtkexodusII/src/ex_utils.c
++++ b/ThirdParty/exodusII/vtkexodusII/src/ex_utils.c
+@@ -1770,7 +1770,6 @@ void ex__compress_variable(int exoid, int varid, int type)
+         */
+ 
+         /* const int NC_SZIP_EC = 4; */ /* Selects entropy coding method for szip. */
+-        const int NC_SZIP_NN = 32;      /* Selects nearest neighbor coding method for szip. */
```

Comment:
Can you use an #ifndef instead?

Reply:
Good idea, that's less intrusive. I change it.

Reply:
In this case, I don't need to guard the patch in the build with `has_version`. Do i?